### PR TITLE
feat: Install script for Windows

### PR DESCRIPTION
**Pull request by daffinito**
# Issue

IHEDIAG-311

# Goals

Create an installation script for nrdiag on Windows to facilitate 1 line install and run

# Implementation Details

The script checks to make sure it's being run on Windows and a supported proc architecture (x64, x86, or arm64). 

It installs nrdiag to the pwd

The installation goes to a tmp dir then is moved to the final destination. 

If the zip file for the current version already exists in TEMP, it will skip re-downloading it.

If there are any existing `nrdiag.exe` or `nrdiag_${plat}.exe` files in the pwd, they will be deleted first.

The binary is just named nrdiag.exe in the end, it removes the _x64 or _arm64.

It only installs the latest version, version is not configurable.

# How to Test

Run the command below, reach out to me for the URL in slack

```
[Net.ServicePointManager]::SecurityProtocol = 'tls12, tls'; (New-Object System.Net.WebClient).DownloadFile("GET_THIS_FROM_ME_IN_SLACK", "$env:TEMP\install.ps1"); & PowerShell.exe -ExecutionPolicy Bypass -File $env:TEMP\install.ps1; $env:NEW_RELIC_LICENSE_KEY='YOUR_STAGING_LICENSE_KEY'; & '.\nrdiag.exe' -a -s dotnet
```

if you try to run the script from a non-Windows host, you should get an error

```
$ pwsh ./scripts/install/install.ps1 

Starting installation

This script is designed for Windows. Please see our documentation for other options to install the Diagnostics CLI on this host.

https://docs.newrelic.com/docs/new-relic-solutions/solve-common-issues/diagnostics-cli-nrdiag/diagnostics-cli-nrdiag/
```

### CODE REVIEW

File: scripts/install/install.ps1 [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+if (Test-Path -Path ".\nrdiag_${arch}.exe" -PathType Leaf) {
+    Write-Output "Removing existing nrdiag_${arch}.exe binary"
+    Remove-Item ".\nrdiag_${arch}.exe"
+}
+
+# Check for existing zip
+if (-not(Test-Path -Path "$env:TEMP\nrdiag_${version}_Windows_${arch}.zip" -PathType Leaf)) {
+    # Download Zip if it doesn't already exist in temp
+    try {
+        $WebClient.DownloadFile("${base_url}/nrdiag/nrdiag_${version}_Windows_${arch}.zip", "$env:TEMP\nrdiag_${version}_Windows_${arch}.zip");
```
Comment (jaffinito) [inline scripts/install/install.ps1:65]: There isn't a "windows" or "x64" version on the download currently

Comment (daffinito) [reply scripts/install/install.ps1:65]: They'll get created on download.newrelic.com when we do the release after this is merged

Comment (daffinito) [reply scripts/install/install.ps1:65]: The script that runs during our release was updated also as part of this feature work 

https://github.com/newrelic/newrelic-diagnostics-cli/pull/150

Comment (jaffinito) [reply scripts/install/install.ps1:65]: Ahh, okay.

**Review by jaffinito — APPROVED** [review]
Code ran fine, minus the missing files misunderstanding.

**Pull request merged**